tinyrpc/protocols/__init__.py

```diff
@@ -324,3 +325,16 @@ def create_batch_request(
         :rtype: :py:class:`RPCBatchRequest`
         """
         raise NotImplementedError()
+
+
+def default_id_generator(start: int = 1) -> Generator[int, None, None]:
```

Comment:
This seems a little out of place, but I can't think of a more appropriate spot for it. 

Any suggestions?

Reply:
I think it is the most reasonable place for it.